source/includes/code-examples/FindMany.cs

```diff
@@ -0,0 +1,98 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
+
+namespace CSharpExamples.UsageExamples;
+
+public class FindMany
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        // Find multiple documents using builders
+        WriteLine("Finding documents with builders...:");
+        FindMultipleRestaurantsBuilderSync();
+
+        // Extra space for console readability 
+        WriteLine();
+
+        // Find multiple documents using LINQ
+        WriteLine("Finding documents with LINQ...:");
+        FindMultipleRestaurantsLINQSync();
+
+        WriteLine();
+
+        // Find All restaurants
+        WriteLine("Finding all documents...:");
+        FindAllRestaurantsSync();
+    }
+
+    private static void FindMultipleRestaurantsBuilderSync()
+    {
+        // start-find-builders-sync
+        var filter = Builders<Restaurant>.Filter
+            .Eq(r => r.Cuisine, "Pizza");
+
+        var restaurants = _restaurantsCollection.Find(filter).ToList();
+        // end-find-builders-sync
+
+        WriteLine("Number of documents found: " + restaurants.Count);
+    }
+
+    private static void FindMultipleRestaurantsLINQSync()
+    {
+        // start-find-linq-sync
+        var query = _restaurantsCollection.AsQueryable()
```

Comment:
the result of this line is not a query. It's already restaurants

Reply:
Better if your sync and async code will be the same. Here you're adding printing, but that doesn't happen here:  https://github.com/mongodb/docs-csharp/pull/51/files#diff-21844f77d63bb9dceb89de54218ec0150222045850393b20081c6c6dee52f3e4R40